src/Salt/UserBundle/Repository/OrganizationRepository.php

```diff
@@ -27,4 +27,20 @@ public function addNewOrganization($organizationName) {
 
         return $org;
     }
+
+    /**
+     * Search all organizations which contains passed string.
+     *
+     * string|null $search
+     *
+     * @return array
+     */
+    public function findOrg($search): array
+    {
+        $qb = $this->createQueryBuilder('org');
```

Comment:
Codacy was complaining about the variable length, maybe replace `$qb` with `$query`.